.github/workflows/build-npm-package-action.yml

```diff
@@ -25,25 +25,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           ref: ${{ inputs.ref }}
 
       - name: Setup Node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: 16.x
+          node-version: 20
```

Comment:
Do we need to pass `node-version` here? In other places we don't.

Reply:
I think that having more control over the crucial CI that builds the package is important to ensure a reproducible environment. However, in other CIs where node is simply used as a test script runner, its version becomes less important.

Reply:
Okay, this makes perfect sense to me, thanks!